.github/workflows/update-versions.yml

```
@@ -22,8 +22,8 @@ jobs:
           # Get all releases and find the first one starting with 'v'
           RELEASE_INFO=$(curl -s "https://api.github.com/repos/filecoin-project/lotus/releases/latest")
           
-          # Extract the tag name
-          TAG_NAME=$(echo "$RELEASE_INFO" | jq -r .tag_name)
+          # Find first tag that starts with 'v' but not 'miner'
+          TAG_NAME=$(echo "$RELEASE_INFO" | jq -r '[.[] | select(.tag_name | startswith("v") and (contains("miner") | not))][0].tag_name')
```

The latest release endpoint returns an object not an array
```suggestion
          TAG_NAME=$(echo "$RELEASE_INFO" | jq -r 'select(.tag_name | startswith("v") and (contains("miner") | not)) | .tag_name')
```